jfix-stdlib-id-generator-jmh/build.gradle.kts

```diff
@@ -19,19 +15,19 @@ jmh{
 
 dependencies {
 
-    compile(project(":jfix-stdlib-id-generator"))
+    implementation(project(":jfix-stdlib-id-generator"))
 
     /**
      * Runtime
      */
-    compile(Libs.slf4j_api)
-    compile(Libs.kotlin_jdk8)
-    compile(Libs.kotlin_stdlib)
-    compile(Libs.kotlin_reflect)
-    compile(Libs.slf4j_simple)
-    compile(Libs.jmh)
-    compile(Libs.jmh_generator_ann)
-    compile(Libs.jmh_generator_bytecode)
+    implementation(Libs.slf4j_api)
+    implementation(Libs.kotlin_jdk8)
```

Comment:
![image](https://user-images.githubusercontent.com/9992425/83259910-8e840c00-a1c1-11ea-84d9-4828a696dac9.png)
As I see, slf4j and kotlin are not used here

Reply:
removed redundant dependencies + changed configuration of `jmh` and `jmh_generator_bytecode` dependencies to `runtimeOnly`, because we don't use these libraries directly from  code.

To make sure this works, I ran a JMH tests in the local environment